src/main/java/edu/ucar/unidata/rosetta/controller/TemplateController.java

```diff
@@ -96,12 +355,14 @@ private String getUploadDir() {
      * @param model The Model object to be populated by Resources.
      * @return The 'create' ModelAndView containing the Resource-populated Model.
      */
+    /*
     @RequestMapping(value = "/create", method = RequestMethod.GET)
     public ModelAndView createTemplate(Model model) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)